src/components/services-view/services-view.controller.ts

```diff
@@ -55,7 +57,8 @@ export class ServicesViewController implements angular.IController {
     this.debounceResize = _.debounce(() => this.resizeExpansion(false), 50, { maxWait: 250 });
     $(window).on('resize.services', this.debounceResize);
 
-    this.ctrl.currentFilter = this.ctrl.currentSubFilter = 'all';
+    this.ctrl.currentFilter = this.$location.$$search.category || 'all';
+    this.ctrl.currentSubFilter = this.$location.$$search.category && this.$location.$$search.subcategory;
```

Comment:
We should use `$location.search()` to get the current values instead of accessing `$$search` directly.

$location.search()

Reply:
Gotcha thanks :+1: